content/blog/larkfid-twoja-baza-wiedzy/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Projekt LARKFid, twoja baza wiedzy o regionie
+date: 2023-08-21T09:20:21+02:00
+draft: false
+github_link: "https://github.com/knmlprz/"
+author: "Piotr Krawiec"
+tags:
+  - projekt
+  - python
+image: /blog/larkfid-twoja-baza-wiedzy/1.png
+---
+
+## Wstęp
+
+W ramach naszego koła naukowego czasami organizujemy całoroczne hackathony, nasi członkowie zbierają się w grupy i przygotowują aplikacje, które konkurują o tytuł najlepszej. Aplikacja, którą wam opiszę — LARKFid powstała właśnie na takim hackathonie.
+
+Naszym celem, jako grupy uczestniczącej w Hackathonie, było stworzenie aplikacji zbierającej, deduplikującej, przetwarzającej i umożliwiającej dostęp do informacji z regionu tj. województwa podkarpackiego. Informacje te mogłyby być dostosowane do klienta.
+
+Zbieranie danych z regionu polega na przeszukiwaniu stron internetowych, forów i grup dyskusyjnych z regionu w poszukiwaniu informacji. Przykładami takich informacji mogą być: ogłoszenia o zaginionych psach, dzikach chodzących po ulicach itp. - w skrócie, wszystko.
+
+Jak możesz się domyślać lub nie, wiele z tych źródeł będzie powielało te same informacje, stąd aplikacja powinna móc je deduplikować.
+
+> Deduplikacja danych to krok, który ma połączyć/usunąć te same informacje, pochodzące z różnych źródeł np. z dwóch portali piszących o tym samym wydarzeniu.
```

Comment:
To zdanie mi się nie klei, ale nie mam teraz pomysłu

Reply:
Usunąłem je